Makefile

```diff
@@ -322,8 +325,7 @@ celeryd:
 	@if [ "$(VENV_BASE)" ]; then \
 		. $(VENV_BASE)/awx/bin/activate; \
 	fi; \
-	$(PYTHON) manage.py celeryd -l DEBUG -B -Ofair --autoreload --autoscale=100,4 --schedule=$(CELERY_SCHEDULE_FILE) -Q tower_scheduler,tower_broadcast_all,$(COMPOSE_HOST),$(AWX_GROUP_QUEUES) -n celery@$(COMPOSE_HOST)
-	#$(PYTHON) manage.py celery multi show projects jobs default -l DEBUG -Q:projects projects -Q:jobs jobs -Q:default default -c:projects 1 -c:jobs 3 -c:default 3 -Ofair -B --schedule=$(CELERY_SCHEDULE_FILE)
+	celery worker -A awx -l DEBUG -B -Ofair --autoscale=100,4 --schedule=$(CELERY_SCHEDULE_FILE) -Q tower_scheduler,tower_broadcast_all,$(COMPOSE_HOST),$(AWX_GROUP_QUEUES) -n celery@$(COMPOSE_HOST)
```

Comment:
@jangsutsr @wwitzel3 

https://github.com/ansible/awx/blob/devel/tools/docker-compose/supervisor.conf#L7

Reply:
Good catch, confirmed using supervisor in devel will constantly have
```
...
awx_1        | 2017-11-09 20:10:52,616 INFO spawned: 'celeryd' with pid 614
awx_1        | 2017-11-09 20:10:53,620 INFO success: celeryd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1        | Unknown command: 'celeryd'
awx_1        | Type 'manage.py help' for usage.
...
```